### DISCUSSION TITLE
Ability to hide() by default

### DISCUSSION THREAD

**Discussion by pokmot** [spatie/ray#83] [Ideas]
Hi, I would love to be able to use hide() by default, and use unhide() on items I want to be fully visible.

I am dumping several models that *could* be important, and take up a lot of space with a lot of scrolling, but what I want to check first is the values towards the bottom. Then I can check the relevant model. I end up using hide() on most of my entries. It would be simpler to set autohide() at the top then use unhide() on the items I want to be fully visible.

**Comment by freekmurze** [comment]
I like the idea, and I'd accept a PR that adds this in a clean way. If none such a PR comes in, I'll take a look myself in a while.
